cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/Checker.java

```diff
@@ -89,20 +88,79 @@ protected void init(Map<String, CommandSimulator> simulators) {
                             public List<SimulationState> getLoopStates() {
                                 return null;
                             }
-                        }, new SimulationState(functionBody, 0));
+                        }, new SimulationState(functionBody, 0, suspiciousPoints));
 
                         return functionSimulator.simulate(ctx, state, command);
                     });
+
+                    simulators.put("set", new CommandSimulator() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this anonymous class number of lines from 27 to at most 20, or make it a named class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1188)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this anonymous inner class a lambda [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1604)